dags/aps/aps.py

```diff
@@ -27,22 +31,25 @@ def save_json_in_s3(dates: dict, repo: IRepository = APSRepository()):
         rest_api = APSApiClient(
             base_url=os.getenv("APS_API_BASE_URL", "http://harvest.aps.org")
         )
-        articles_metadata = str.encode(
-            json.dumps(rest_api.get_articles_metadata(parameters))
-        )
-        return save_file_in_s3(data=articles_metadata, repo=repo)
+        articles_metadata = rest_api.get_articles_metadata(parameters)
+        if articles_metadata is not None:
+            articles_metadata = str.encode(
```

Comment:
I still don't understand why encoding :/

Reply:
Otherwise, I'm getting an error, when I'm trying to write it to the file:
` TypeError: a bytes-like object is required, not 'str'`
Code, where the error is raised: [here](https://github.com/cern-sis/workflows/blob/full_aps_proccessing/dags/aps/utils.py#L34)

Reply:
`get_articles_metadata` already returning [json](https://github.com/cern-sis/workflows/blob/e7d33f397a72df5abd87d8cd5bdb4ed7c7a30111/dags/aps/aps_api_client.py#L26) why you don't save it directly?

Reply:
solved ☺️